src/Command/FixContextCommand.php

```diff
@@ -18,12 +18,15 @@
 use Sonata\ClassificationBundle\Model\ContextInterface;
 use Sonata\ClassificationBundle\Model\ContextManagerInterface;
 use Sonata\ClassificationBundle\Model\TagManagerInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+#[AsCommand(name: 'sonata:classification:fix-context', description: 'Generate the default context if none defined and attach the context to all elements')]
```

Comment:
I think this is the correct way to avoid this deprecation, wdyt @VincentLanglet ? In SonataAdminBundle we fixed it different but to me this is more correct since the attribue only gets used on Symfony 6.1, and the older versions can maintain what they have for lazy commands.

Reply:
I prefer this solution over https://github.com/sonata-project/SonataBlockBundle/pull/1075 or (the SonataAdmin PR).
So the attribute solve the deprecation even if the static property is still used ?

Is there a reason to not doing this on Admin/BlockBundle or can we update the code too ?

Reply:
We can update that code too.